packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -74,7 +74,7 @@ export class DocumentDeltaConnection
 
 	private _details: IConnected | undefined;
 
-	private trackLatencyTimeout: number | undefined;
+	private trackLatencyTimeout: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Huh, how was this passing type checks before when we were assigning to it?

Reply:
Not sure - same as the change below, the typing changes tickled TS in some way that made it realize there was bad typing in this file.  Kind of spooky that they weren't caught earlier, but after investigating I think both were correctly identified as errors.